src/Symfony/Component/Messenger/Transport/RedisExt/Connection.php

```diff
@@ -141,6 +159,10 @@ public function reject(string $id): void
 
     public function add(string $body, array $headers): void
     {
+        if ($this->autoSetup) {
```

Comment:
I think we should in some case avoid to call setup on every add or get, maybe set `$this->autoSetup` to false in setup() function?

Reply:
👍 done